plugins/urlscan/komand_urlscan/actions/search/action.py

```diff
@@ -54,10 +81,12 @@ def run(self, params={}):
             url = f'{self.connection.server}/search/?{"&".join(query_params)}'
             self.logger.info(url)
 
-            try:
-                response = requests.get(url, headers=self.connection.headers)
-            except Exception as e:
-                raise PluginException(cause="Something went wrong during the request.", assistance=e)
+            response = requests.get(url, headers=self.connection.headers)
+            json_response = json.loads(response.text)
+            if json_response.get("status") == 400:
+                raise PluginException(
+                    cause="400 Error - Invalid request.", assistance=f"{json_response.get('message')}"
```

Comment:
Can we use the standard 400 Bad request message for cause and assistance here and the response message can be returned in the data field?

We should also retain catching all exceptions and raising a plugin exception.